tests/unit/amo/components/TestUserProfileEdit.js

```diff
@@ -125,58 +128,111 @@ describe(__filename, () => {
     expect(root.find(UserProfileEditPicture)).toHaveLength(1);
   });
 
-  it('dispatches fetchUserAccount action if username is not found', () => {
+  it('dispatches fetchUserAccount and fetchUserNotifications actions if username is not found', () => {
     const { store } = signInUserWithUsername('tofumatt');
     const dispatchSpy = sinon.spy(store, 'dispatch');
 
     const username = 'i-am-not-tofumatt';
     const root = renderUserProfileEdit({ params: { username }, store });
 
+    sinon.assert.callCount(dispatchSpy, 2);
     sinon.assert.calledWith(dispatchSpy, fetchUserAccount({
       errorHandlerId: root.instance().props.errorHandler.id,
       username,
     }));
+    sinon.assert.calledWith(dispatchSpy, fetchUserNotifications({
+      errorHandlerId: root.instance().props.errorHandler.id,
+      username,
+    }));
   });
 
-  it('does not dispatch fetchUserAccount action if there is no username', () => {
-    const { store } = signInUserWithUsername('tofumatt');
+  it('only dispatches fetchUserNotifications action when the current logged-in user is being edited', () => {
```

Comment:
Nit: I found this description a bit confusing, until I figured it out. I interpreted the `only` as "it will only do this when", like as a condition, whereas I now realize you mean "it will only dispatch fetchUserNotifications and not fetchUserAccount". Do you think you can find a way to word it that is less ambiguous?

Reply:
Please, do let me know every time my English is not correct 🤗 

I will rephrase this description.